app/routes/crate/version.js

```diff
@@ -25,14 +27,33 @@ export default class VersionRoute extends Route {
     let version;
     let requestedVersion = params.version_num;
     if (requestedVersion) {
-      version = versions.find(version => version.num === requestedVersion);
+      version =
+        crate.loadedVersionsByNum.get(requestedVersion) ??
+        (await this.store
+          .queryRecord('version', {
+            name: crate.id,
+            num: requestedVersion,
+          })
+          .catch(() => {
+            // ignored
+          }));
```

Comment:
if we still use `crate.loadVersionsTask` above, what is the reason for this fallback? 🤔

Reply:
Yes, that's a completely fair question. We still have dependencies on the versions (especially the version count), which makes the fallback seem unnecessary. The fallback would become valuable if we could load the page without blocking to wait for versions.